staging/src/k8s.io/client-go/util/certificate/certificate_manager.go

```diff
@@ -349,30 +327,34 @@ func (m *manager) rotateCerts() (bool, error) {
 	}
 
 	m.updateCached(cert)
-	m.setRotationDeadline()
-	m.forceRotation = false
 	return true, nil
 }
 
-// setRotationDeadline sets a cached value for the threshold at which the
+// nextRotationDeadline returns a value for the threshold at which the
 // current certificate should be rotated, 80%+/-10% of the expiration of the
 // certificate.
-func (m *manager) setRotationDeadline() {
+func (m *manager) nextRotationDeadline() time.Time {
+	// forceRotation is not protected by locks
```

Comment:
why?

Reply:
I was probably being overzealous, the only thing protected by certAccessLock is... cert.  Before, there was a set of local vars that were called in two different goroutines, now everything is in a single goroutine.  I can remove the comment.